### DISCUSSION TITLE
How to lower CPU use?

### DISCUSSION THREAD

**Discussion by dbrowdy** [jef/streetmerchant#1672] [Help]
This program is pegging my CPU at 100%, even after a fresh reboot with nothing else running.  I have an i5-8500 which is a couple years old but not terrible.  I see about 20+ instances of Chromium running at a time. 

Anyone know a way to throttle the CPU use?  I tried adding delays to PAGE_SLEEP_MIN and MAX and it didn't seem to help much.  Am I just searching for too much at a time?  It looking for 3070,3080,5600X and 5800X at about a dozen US stores.

**Comment by footloops** [comment]
It's probably because your searching for a lot in multiple different stores. I'm only searching for the 5600x in 3 stores and have an avg of 5-10% cpu usage.

EDIT: I'm using an i7 8550u

**Reply by THarutyunyan** [reply]
Can you please tell how to search for only 1 product? thanks in advance.

**Comment by wofnull** [comment]
The problem you have indeed is to much stores / articles you are searching for:
-> Every Item gets checked after another in one store, in parralel, streetmerchant does this on any site you entered.
Thus said, the only way to lower your performance hit is, to reduce the amount of sites you are searching through
